Makefile.containers

```diff
@@ -32,4 +32,24 @@ else
 CONTAINER_COMPOSETOOL = $(error Neither docker-compose nor podman-compose found)
 endif
 
-CONTAINER_VARS = CONTAINER_CLITOOL CONTAINER_COMPOSE CONTAINER_BUILDENV
+ifeq ($(OS_RELEASE), ubuntu)
+CONTAINER_OS_TYPE=deb
+else ifeq ($(OS_RELEASE), debian)
+CONTAINER_OS_TYPE=deb
+else ifeq (${OS_RELEASE}, alpine)
+CONTAINER_OS_TYPE=apk
+else ifeq (${OS_RELEASE}, freebsd)
+CONTAINER_OS_TYPE=pkg
```

Comment:
@dhurley Is this required or not? As I understand we do not have plans to support `freebsd` images
```suggestion
```

Reply:
not required. We dont have plans for freebsd images